Use open_port to spawn epmd

filmor opened the pull request:
This ensures that spaces in the path are treated correctly.

### Description

Exchange `os:cmd` for a call to `open_port({spawn_executable ...`, adjusted from https://stackoverflow.com/a/2249387/542190.

Fixes https://github.com/erlang-ls/vscode/issues/24.

robertoaloi commented:
@richcarl Your answer is from 2010. Do you know if this is still the best to get this fix? I'd prefer not to have to fork the code from `os:cmd`.

richcarl commented:
> @richcarl Your answer is from 2010. Do you know if this is still the best to get this fix? I'd prefer not to have to fork the code from `os:cmd`.

spawn_executable is much safer than the general cmd, but as far as I know the only way of using it is by rolling your own. This piece of code is however way simpler than the actual os:cmd implementation, so not much of a worry.

filmor commented:
Fixed the spec. This is not a "fork" of the original `os:cmd`, I just simplified the version given in the SO answer as just to call `epmd -daemon` we need a lot less processing (in particular, the output is completely irrelevant as it is empty).

robertoaloi commented on src/els_db.erl:
```diff
@@ -12,7 +12,8 @@
         , tables/0
         , transaction/1
-        , write/1
+        , write/1,
```
The project currently uses a comma-first indentation. Things may change in the future, for now please follow the guidelines at https://github.com/erlang-ls/erlang_ls/blob/master/.github/CONTRIBUTING.md

robertoaloi replied:
Also, why exporting it?

filmor replied:
Ah, sorry, I did this just for testing, that's also why I didn't adhere to the guidelines. Will fix.

robertoaloi commented:
Thanks!
